src/main/resources/schema.sql

```diff
@@ -0,0 +1,17 @@
+create table accountTable
+(
+  id   UUID        NOT NULL,
+  name VARCHAR(50) NOT NULL,
+  PRIMARY KEY (id),
+);
+
+create table transactionTable
```

Comment:
use `transaction` as a table name? I think you should be able to escape the reserved work using backticks `.